Redesign the Learn Landing Page (#2473)
ballerina-platform/ballerina-dev-website: documentation.md

---
layout: ballerina-learn-landing-page
title: Ballerina Documentation
description: Ballerina is a comprehensive language that is easy to grasp for anyone with prior programming experience. Start learning with the material below.
keywords: ballerina, learn, documentation, docs, programming language
permalink: /learn-beta2/
intro: Ballerina is a comprehensive language that is easy to grasp for anyone with prior programming experience. Start learning with the material below.
redirect_from:
- /learn-beta2
---
## Getting Started

### [Language Introduction Video](https://www.youtube.com/watch?v=My_uqtHvXV8&t=10s)

> Video presentation of above which people can use as a sample of how to present the one hour deck.

Reviewer: ```suggestion
A video introduction to the Ballerina Programming Language and what's new in Swan Lake.
```

Author: Fixed.

### [Language Taste Screencasts](https://www.youtube.com/watch?v=My_uqtHvXV8&t=10s)

> End-to-end video screencasts like the one Manu has done on writing a service.

Reviewer: End-to-end screencasts covering popular use cases that give a taste of the Ballerina language.

Author: Fixed.

### [Language Walkthrough Slides](http://localhost:4000/learn/language-concepts/Ballerina_Language_Presentation-2021-03-08.pdf)

> Slide deck explaining the language.

Reviewer: ```suggestion
A comprehensive reference slide deck explaining the language.
```

### [Language Walkthrough Video](https://www.youtube.com/watch?v=My_uqtHvXV8&t=10s) 

> Video recording of the language slide deck.

Reviewer: ```suggestion
A video recording of the language slide deck.
```

### [Learn by Example](/learn/by-example/)

> Guided path to learn the language by a series of examples.

Reviewer: ```suggestion
A guided path to learn the language through a series of examples.
```

## Concepts

### [Why Ballerina](/learn/why-ballerina/)

Explains the unique aspects of Ballerina.

### [User Guide](/learn/user-guide/)

> Series of guides on various things you need to do during the Ballerina development cycle.

Reviewer: ```suggestion
A series of guides on various things you need to do during the Ballerina development cycle.
```

## References

### [Language Introduction Slides](http://localhost:4000/learn/language-concepts/Ballerina_Language_Presentation-2021-03-08.pdf)

> One hour presentation on the language that 3rd parties can use to talk about the language.

Reviewer: ```suggestion
Presentation slides on the Ballerina language that anyone can use to talk about the language.
```

Author: Fixed.

### [Language Guide](/learn/language-concepts/)

> Prose extracted from slide deck and videos, with expanded examples.

Reviewer: ```suggestion
An elaborate textual guide to the Ballerina language. 
```

### [Library Documentation](/learn/api-docs/)

All library API docs.

### [Reference Guide by Examples](/learn/by-example/)

> Series of examples that serve as a reference guide for each language construct & concept.

Reviewer: ```suggestion
A series of examples that serve as a reference guide for each language construct & concept.
```

Author: Fixed.
